MoltenVK/MoltenVK/GPUObjects/MVKRenderPass.mm

```diff
@@ -327,15 +327,11 @@ static uint32_t getNextViewMaskGroup(uint32_t viewMask, uint32_t* startView, uin
 			mtlTexDesc.textureType = MTLTextureType2DMultisample;
 			mtlTexDesc.sampleCount = sampleCount;
 		}
-#if MVK_IOS
-		if ([_renderPass->getMTLDevice() supportsFeatureSet: MTLFeatureSet_iOS_GPUFamily1_v3]) {
+		if (mvkOSVersionIsAtLeast(10.0, 11.0, 14.0)) {
```

Comment:
This will break on macOS 11 with a non-Apple GPU.

Reply:
Then again, we shouldn't even hit that codepath on macOS 11, since it supports render passes with no render targets. It's why I didn't bother updating that for Apple Silicon.

Reply:
Another good catch. 

Actually in review, I realized that `mvkOSVersionIsAtLeast(macOS, iOS, macCat)` won't actually work, so I've reverted it. 

It turns out that `mvkOSVersion()` reports the same value for macOS and MacCat (eg. 11.0001), and reports the same value for iOS and iOS simulator (eg. 14.2), so there is no distinct MacCat OS version to act on.

Reply:
Also. The original code was just checking for iOS anyway, and is not affected by Mac Cat, so I've reverted it to the original.

I think I made this original change early on when I was originally thinking of MacCat as a "variation of iOS", instead of the current "variation of macOS", and missed backing it out. 🤦🏻‍♂️